x-pack/plugins/enterprise_search/public/applications/app_search/components/result/types.ts

```diff
@@ -17,7 +17,6 @@ export interface FieldValue {
 
 export interface ResultMeta {
   id: string;
-  scopedId: string;
```

Comment:
It's still there in the data being sent from the server though right? No strong feelings either way, but I wonder if we should leave it in because it'll be used by curations at some point, or only add it back in then?

Reply:
It's not.